docs/get-started/lotus/installation.md

```diff
@@ -30,7 +30,11 @@ These are the minimal requirements to run a Lotus node. [Hardware requirements f
 
 The following instructions are specific to Linux installations. Head to the [macOS](#macos) section if you want to install Lotus on a Mac.
 
-Users can install Lotus on Linux using the [Snap package manager](#snap-package-manager), [AppImages](#appimage), or by [building from source](#build-from-source).
+Users can install Lotus on Linux using the 
```

Comment:
[formatting] If you split into a bullet list, the sentence before that should be a complete sentence, and end with `:` (e.g. "Users can install Lotus on Linux using any of these methods:". Or you can leave as a single sentence.